### DISCUSSION TITLE
Youtube broadcast / normal video setting

### DISCUSSION THREAD

**Discussion by Yucie** [smplayer-dev/smplayer#1004] [Q&A]
I want to asking for youtube broadcast and normal video setting with yt-dlp.

When i watch youtube boardcasting video. I need to set use adaptive streams that let the video and sound synchronize. If not the video and sound will have at least 5 second delay or more.
![test1](https://github.com/smplayer-dev/smplayer/assets/12384865/1ebaa0c9-82d4-4797-8ed0-3f0c83c3c3f0)

However, if i set use adaptive streams, normal video can not play and show download new yt-dlp. After yt-dlp downloaded. the link still show error and require dl again.
![test2](https://github.com/smplayer-dev/smplayer/assets/12384865/9e9c7568-7567-4632-9ad5-f165d824e567)
If i want to watch normal video, it need to unset use adaptive streams.

Before, i unset use adatpive stream that can play boardcast video and normal video correctly.
This problem occur from last week, i want to know the setting for youtube video that can play broadcast video and normal video correctly.

Thank you.

**Comment by smplayer-dev** [comment]
Does it make a difference if you select "mpv" in "support for video sites"?

**Reply by Yucie** [reply]
> Does it make a difference if you select "mpv" in "support for video sites"?

My mulitmedia engine setting is mpv.
![tttte](https://github.com/user-attachments/assets/ed3401a6-3f27-4a0a-bf4a-e1a067ac9b1e)

![ttte2](https://github.com/user-attachments/assets/f87f9561-cb35-4979-ab5c-e42258836208)


If not, youtube video cant play with smplayer.

Last week, same setting that can play boardcasting video and normal video.
After, youtube boardcasting video cant play (unset use adaptive stream ) correctly. Sound will delay.
If i turn on use adaptive stream, boardcasting video will play correctly. 
However, normal video will show cant find youtube link and popup warning to download new yt-dlp.exe

**Comment by smplayer-dev** [comment]
I mean this option:
![Captura desde 2024-07-13 10-54-51](https://github.com/user-attachments/assets/b3841e7d-9dff-451a-9a25-a3c99376af29)

**Reply by Yucie** [reply]
![ttea](https://github.com/user-attachments/assets/d402d2c3-b3ed-425c-b6d3-0dcfda757bc8)

Work OK!

**Reply by Yucie** [reply]
Problem from mpv setting

https://github.com/user-attachments/assets/39738738-b621-4d10-a067-37f61f721f28
I attach this video from my mpv testing. (I dont know other people can watch it or not)

When i set mpv for support site. If i shift the normal youtube video, the video will freeze or become sound only.
However, i reset it to  auto for support site, the video can shift and did not freeze or sound only.

If i set to mpv, video can watch but cant shift time.
If i set to auto, video can play with time shift correctly.

As the result, i set auto set for site.
